calico_kubernetes/calico_kubernetes.py

```diff
@@ -9,13 +9,18 @@
 from subprocess import check_output, CalledProcessError, check_call
 import requests
 import sh
+import logging
 from netaddr import IPAddress, AddrFormatError
+from calico_logging import config_logger
 from pycalico import netns
 from pycalico.datastore import IF_PREFIX, DatastoreClient
 from pycalico.util import generate_cali_interface_name, get_host_ips
 from pycalico.ipam import IPAMClient
 from pycalico.datastore_errors import PoolNotFound
 
+LOG_DIR = '/var/log/calico/kubernetes'
```

Comment:
I'd move LOG_DIR to calico_logging.  It doesn't need to be configurable from this file.